src/eosjs-numeric.ts

```diff
@@ -365,7 +365,18 @@ export function stringToPrivateKey(s: string): Key {
     if (s.substr(0, 7) === 'PVT_R1_') {
         return stringToKey(s.substr(7), KeyType.r1, privateKeyDataSize, 'R1');
     } else {
-        throw new Error('unrecognized private key format');
+        // todo: Verify checksum: sha256(sha256(key.data)).
+        //       Not critical since a bad key will fail to produce an
+        //       invalid signature anyway.
```

Comment:
Should be "fail to produce valid signature"

Reply:
fixed